src/scripts/main.js

```diff
@@ -1,3 +1,263 @@
 'use strict';
 
-// write code here
+const MIN_AGE = 18;
+const MAX_AGE = 100;
+const pageBody = document.querySelector('body');
+const tableHeader = document.querySelector('thead');
+const tableHeaderCells = tableHeader.querySelector('tr');
+const tableBody = document.querySelector('tbody');
+let isSecondClick = false;
+const toNumber = (string) => {
+  if (typeof string === 'string' && string) {
+    const clearNum = string.replace(/[$, ]/g, '');
+
+    return parseInt(clearNum, 10);
+  }
+};
+let clickedSortTable;
+
+tableHeaderCells.addEventListener('click', (e) => {
+  const headerChildren = [...tableHeaderCells.children];
+  const columnName = e.target.closest('th');
+
+  if (columnName !== null) {
+    const sortedColumn = headerChildren.indexOf(columnName);
+    const columnsToSort = [...tableBody.children];
+
+    if (clickedSortTable === columnName) {
+      isSecondClick = !isSecondClick;
+    } else {
+      isSecondClick = false;
+    }
+
+    clickedSortTable = columnName;
+
+    const columnsSorted = columnsToSort.sort((a, b) => {
+      const elemA = a.children[sortedColumn].textContent;
+      const elemB = b.children[sortedColumn].textContent;
+
+      const aToNum = toNumber(elemA);
+      const bToNum = toNumber(elemB);
+
+      if (Number.isNaN(aToNum)) {
+        return isSecondClick
+          ? elemB.localeCompare(elemA)
+          : elemA.localeCompare(elemB);
+      }
+
+      return isSecondClick ? bToNum - aToNum : aToNum - bToNum;
+    });
+
+    tableBody.textContent = '';
+
+    columnsSorted.forEach((el) => {
+      tableBody.appendChild(el);
+    });
+  }
+});
+
+tableBody.addEventListener('click', (e) => {
+  const rowsChildren = [...tableBody.children];
+  const columnName = e.target.closest('tr');
+
+  rowsChildren.forEach((row) => row.removeAttribute('class'));
+  columnName.classList.add('active');
+});
+
+const formContainer = document.createElement('form');
+const nameLabel = document.createElement('label');
+const nameInput = document.createElement('input');
+const positionLabel = document.createElement('label');
+const positionInput = document.createElement('input');
+const officeLabel = document.createElement('label');
+const officeChoose = document.createElement('select');
+const ageLabel = document.createElement('label');
+const ageInput = document.createElement('input');
+const salaryLabel = document.createElement('label');
+const salaryInput = document.createElement('input');
```

Comment:
Create a helper function to encapsulate the repetitive element creation logic. For example, you can create a function called `createFormField` and keep there logic regarding creation and applying some attributes.